### PR TITLE
Updates delete process in license FAQ

### PR DESCRIPTION
**Pull request by matt1hathcock**
- updated langauge in html file for license FAQ

### CODE REVIEW

File: site/license-faq.html [hunk 1]
```diff
@@ -573,7 +573,7 @@ <h3 id="32">I'm not using FusionAuth any more. How can I close my account and ca
               Once you have deleted all of your deployments and downgraded to Community Edition, your subscription will be canceled and you won’t be billed going forward.
             </p>
             <p>
-              If you have any questions regarding any of these steps, please contact sales@fusionauth.io. However, please note that we cannot do this for you.
+              If you would then like your user information completely removed please contact <a href="mailto:sales@fusionauth.io">sales@fusionauth.io</a>. Historical payment information must be maintained and does not contain any PII.
```
Comment (mooreds) [inline site/license-faq.html:576]: I would point people to the contact form here rather than the sales email alias.

**Comment by mooreds** [comment]
One comment, otherwise looks good.

**Review by mooreds — APPROVED** [review]
One small change requested.

File: site/license-faq.html (hunk 1 above)
Comment (matt1hathcock) [reply site/license-faq.html]: updated in latest commit

**Pull request merged**